frontend/riskofbias/constants.js

```diff
@@ -80,15 +80,25 @@ const NA_KEYS = [10, 20],
     BIAS_DIRECTION_UNKNOWN = 0,
     BIAS_DIRECTION_UP = 1,
     BIAS_DIRECTION_DOWN = 2,
+    BIAS_DIRECTION_CHOICES = {
+        [BIAS_DIRECTION_UNKNOWN]: "? (Unknown/not specified)",
+        [BIAS_DIRECTION_UP]: "⬆ (Away from null)",
+        [BIAS_DIRECTION_DOWN]: "⬇ (Towards null)",
+    },
+    BIAS_DIRECTION_COMPACT = {
+        [BIAS_DIRECTION_UNKNOWN]: "",
+        [BIAS_DIRECTION_UP]: "⯅",
```

Comment:
use these instead? ▲ ▼

Reply:
if we keep running into issues with this we can use svg-based triangles that will work on anyone's browser

Reply:
Changed the triangles. So now all of our directional characters are from unicode version 4 instead of 7, so I imagine we'll find these in a bunch more font packs.